public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchLogsQueryRunner.ts

```diff
@@ -325,7 +326,7 @@ export class CloudWatchLogsQueryRunner extends CloudWatchRequest {
 
   getLogRowContext = async (
     row: LogRowModel,
-    { limit = 10, direction = 'BACKWARD' }: RowContextOptions = {},
+    { limit = 10, direction = LogRowContextQueryDirection.Backward }: LogRowContextOptions = {},
```

Comment:
I can't decide this `backwardS` is actually the correct word for this.

Reply:
I think they're interchangeable, but "backward" is more American English 🇺🇸